nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -654,7 +658,7 @@ in
               The name of the S3 bucket.
             '';
           };
-          autocreate = mkOption {
+          verify_bucket_exists = mkOption {
             type = types.bool;
```

Comment:
This should probably default  to true, since that is the default for the implementation as well.

Reply:
I was thinking about that but wasn't totally convinced to do so as i'm not really sure wheter this actually is a sensible default. I've changed it now. Probably better to have the nextcloud default and forcing people to decide by not setting a default (as it stood) is probably also unnecessary.